include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3038,8 +3061,6 @@ struct OrtApi {
    */
   ORT_API2_STATUS(HasValue, _In_ const OrtValue* value, _Out_ int* out);
 
-  /// @}
```

Comment:
It is deleted accidentally?

Reply:
It's not removed, we have it stays right after the lines we added.

Reply:
It was closing region:
`/// \name OrtStatus
/// @{`

Next region starts on next line:
`/// \name OrtKernelContext
/// @{`

And it has its closing `/// @}`.

Actually new methods should have their own doc region/group.

Reply:
Sure, fixed scope by adding /// @{ and /// @} accordingly.